src/components/Card.js

```diff
@@ -15,45 +28,60 @@ export default class Card {
   }
 
   _setEventListeners() {
-    this._cardEl
-      .querySelector(".card__like-button")
-      .addEventListener("click", () => {
-        this._handleLikeIcon();
-      });
+    const likeButton = this._cardEl.querySelector(".card__like-button");
```

Comment:
You've already found `this._likeButton ` in line 73. Please, use it everywhere


`Class fields` (the so-called **class variables** with `this`) are accessible inside any class method. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.